src/Element/Checkbox.php

```diff
@@ -57,16 +57,16 @@ public function setOptions($options)
     {
         parent::setOptions($options);
 
-        if (isset($options['use_hidden_element'])) {
-            $this->setUseHiddenElement($options['use_hidden_element']);
+        if (isset($this->options['use_hidden_element'])) {
```

Comment:
This change is dangerous due to the conditional side effects:
If `$this->options['use_hidden_element']` is set outside of `setOptions()` then change would be unreliably applied depending on whether `setOptions()` was invoked or not.

Reply:
Same goes for all similar changes

Reply:
Disregard as `$this->options` is getting completely replaced by `parent::setOptions()` call rather than getting merged with previously provided options as I expected.